_posts/en/2019-01-20-rbf-in-the-wild.md

```diff
@@ -0,0 +1,243 @@
+
+Electrum allows more advanced control of the fees in the UI. Additionally the dialog provides the capability to mark the transaction as final.
+
+### BRD: Generic Error Messages
+
+![BRD wallet transactions list screen showing a failed transaction screenshot](/img/posts/rbf-in-the-wild/rbf-brd-transaction-list-incoming-bumped.png)
+
+BRD shows a “Failed” label on the replacement transaction. In testing, the “Failed” transaction actually ends up being the transaction that confirms.
+
+## Usability Considerations
+
+A blanket recommendation for wallets, explorers, and other Bitcoin services would be to support Opt-in RBF, BIP125. Most of the services tested in this study do not have any support for RBF. As congestion builds on the network in the future, RBF will be an important tool in the scaling toolbox.
+
+Based on some of the examples above, there are some additional recommendations regarding RBF support.
+
+### Explorers Redirect to Replacement Transaction
+
+In almost all explorers, attempting to view the original transaction after the replacement confirms results in a 404 error. This can be confusing for users that have previously visited the original transaction's page on the explorer site and come back to find it has disappeared.
+
+Explorers should consider 301 redirects from the original to the replaced transaction that eventually confirms. Especially when the original transaction has been viewed by a visitor via that explorer. The redirection should be accompanied with an alert message indicating the original transaction has been replaced by current transaction.
+
+### Persist Original Transactions
+
+This recommendation is similar to previous one, only applied to wallets instead of explorers.
+
+Original transactions should be persisted in the UI, even subtly or under an advanced dialog. When sending a replacement transaction, a user may still want to have a reference to previously attempted, never-to-confirm transaction in their wallet.
+
+### Use RBF Compatible Block Explorers
+
+Wallets that support RBF and do not provide block explorer details within the app itself, should link to an RBF compatible block explorer. This will provide a consistent experience for the user as transactions will not “disappear” when transitioning from the RBF-supporting app to the non-RBF-supporting explorer.
+
+### Link to Block Explorer's Address Page
+
+Wallets/Exchanges should consider linking to the block explorer’s “address” page instead of “transaction” page. This can simplify the fact that transactions might be replaced/disappear from the explorer while also providing a single reliable link where someone can view the transaction.
+
+It is preferable that the explorer implement a redirect scheme similar to the one outlined above, however, this is a more user friendly alternative if that option is not available.
+
+### Label RBF Replaceable Transactions
```

Comment:
Nit: 'Label RBF Replaceable Transactions' -> 'Label Transactions Signaling Opt-in RBF' (same for paragraph below)